assets/css/_logged_in_as.scss

```diff
@@ -0,0 +1,25 @@
+.c-logged-in-as__container {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.c-logged-in-as__icon {
+  height: 32px;
+  width: 32px;
```

Comment:
**suggestion:** (here and a few other places) we usually use `rem` for sizing, except within Leaflet maps because those already use `px` for a lot of the sizing.

Reply:
Ahh good. I was wondering about that, and planning to ask.